src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1092,10 +1096,27 @@ internal double GetPortVerticalOffset(PortModel portModel)
         /// </summary>
         public void RegisterInputPorts()
         {
+            var inputs = new List<PortData>();
+
+            // Old version of input ports registration.
+            // Used InPortData.
+            if (InPortData.Count > 0)
+            {
+                inputs.AddRange(InPortData);
+                Log(String.Concat(Name, ": ", Properties.Resources.DeprecatedPortNamingStyleMessage));
```

Comment:
Let's remove this log. The developer should get compile time warnings, which is sufficient.